stable/test/integration/_exec.pony

```diff
@@ -0,0 +1,85 @@
+use "ponytest"
+use "files"
+use "process"
+use "regex"
+
+actor _Exec
+  new create(
+    h: TestHelper,
+    cmdline: String,
+    notifier: ProcessNotify iso)
+  =>
+    try
+      let args = cmdline.split_by(" ")
+      let path = FilePath(h.env.root as AmbientAuth, "build/release/stable")?
+      let vars: Array[String] iso = []
+      let auth = h.env.root as AmbientAuth
+      let pm: ProcessMonitor = ProcessMonitor(auth, auth, consume notifier,
+        path, consume args, consume vars)
```

Comment:
Just an idea:

I think there is no way to do this currently (correct me if i'm wrong) but wouldn't it be nice to execute the binary with `PWD` in a temporary directory, possibly prepopulated with some `bundle.json` or something else or nothing at all. And afterwards assert on the directory contents in one way or the other.